src/main/plugins/org.dita.html5/xsl/topic.xsl

```diff
@@ -2076,13 +2091,7 @@ See the accompanying LICENSE file for applicable license.
   
   <xsl:template match="*[contains(@class, ' topic/section ')]/*[contains(@class, ' topic/title ')] | 
     *[contains(@class, ' topic/example ')]/*[contains(@class, ' topic/title ')]" name="topic.section_title">
-    <xsl:param name="headLevel">
-      <xsl:variable name="headCount" select="count(ancestor::*[contains(@class, ' topic/topic ')])+1"/>
-      <xsl:choose>
-        <xsl:when test="$headCount > 6">h6</xsl:when>
-        <xsl:otherwise>h<xsl:value-of select="$headCount"/></xsl:otherwise>
-      </xsl:choose>
-    </xsl:param>
+    <xsl:param name="headLevel" as="xs:string" select="'h' || dita2html:get-heading-level(.)"/>
```

Comment:
Again, `||`. But this variable value is not heading level, it's the element name. Remove `h` from it and add the `h` to the `<xsl:element>` for consistency.

Reply:
This bothers me too (that a "level" value stores an element name and not a value. But it's a parameter and not a variable. Doesn't this mean that the convention must be kept for compatibility?